examples/vdpa/main.c

```diff
@@ -179,6 +180,19 @@ start_vdpa(struct vdpa_port *vport)
 			"register driver failed: %s\n",
 			socket_path);
 
+	/* vdpa device should get feature from device and not use builtin_net_driver */
+	ret = rte_vdpa_get_features(vport->dev, &features);
+	if (ret != 0)
```

Comment:
just do

if (ret)

below also

Reply:
all the other code also use this format, if change this,
it will not consistent with other code.